api/app/src/Controller/Ndr/NdrController.php

```diff
@@ -228,4 +229,12 @@ public function update(Request $request, $id)
 
         return ['id' => $ndr->getId()];
     }
+
+    #[Route(path: '/ndr/client/{clientId}', methods: ['GET'])]
```

Comment:
The API endpoint for Client already has an ndr field which is either an NDR or null. This data is included with the client API endpoint response, and available on the Client object on the client/ (confusing!) side of the application.

What I'd suggest is checking whether this field is null, rather than making another request to the API to find out whether the client has an NDR.